### PR TITLE
Nsis osx installer

### LINKED ISSUE #650: OSX NSIS installer
Create an offline NSIS style installer for mac osx that includes the engine, data files and the exporter.

### PR DESCRIPTION
**Pull request by rasberrypai**
Offline NSIS style installer for mac osx.
Gives users the option to install the engine, engine data files and the fusion exporter on mac.

Closes #650

### CODE REVIEW

File: installer/OSX/README.md [hunk 1]
```diff
@@ -1,62 +1,36 @@
...
-6. Run ` appdmg spec.json Synthesis.app `
+    - Remove the data file placeholder ` rm [synthesis/installer/OSX/Assets/payload/Contents/Synthesis/README.md] `	
 
-### Package
+4. Add unzipped exporter files to synthesis/installer/OSX/Exporter/payload/Contents/SynthesisFusionGltfExporter
+  
+    - Remove the exporter file placeholder ` rm [synthesis/installer/OSX/Exporter/payload/Contents/SynthesisFusionGltfExporter/README.md] `	
 
-1. Make a new folder 
+5. Change directories to the osx installer directory ` cd synthesis/installer/OSX `
```
Comment (matthew17754) [inline installer/OSX/README.md:21]: Shouldn't OSX be capitalized?

**Comment by copernicium** [comment]
I don't really have any way to test this.

File: installer/OSX/README.md [hunk 2]
```diff
@@ -1,62 +1,36 @@
...
+2. Zip Synthesis.app
 
-3. Install Node / NPM ` brew install node `
+3. Copy Synthesis.zip to file system ` cp [Synthesis.zip] [synthesis/installer/OSX/App/payload/Contents] `
 
-4. Install AppDMG ` npm i -g appdmg `
+    - Remove the Synthesis.zip placeholder ` rm [synthesis/installer/OSX/App/payload/Contents/README.md] `
 
-5. change directories to the osx/application directory ` cd synthesis/installer/osx/application/ `
+3. Add data files to synthesis/installer/OSX/Assets/payload/Contents/Synthesis
```
Comment (matthew17754) [inline installer/OSX/README.md:13]: Apply `markdown` to this path as well

File: installer/OSX/README.md [hunk 3]
```diff
@@ -1,62 +1,36 @@
...
-6. Run ` appdmg spec.json Synthesis.app `
+    - Remove the data file placeholder ` rm [synthesis/installer/OSX/Assets/payload/Contents/Synthesis/README.md] `	
 
-### Package
+4. Add unzipped exporter files to synthesis/installer/OSX/Exporter/payload/Contents/SynthesisFusionGltfExporter
+  
+    - Remove the exporter file placeholder ` rm [synthesis/installer/OSX/Exporter/payload/Contents/SynthesisFusionGltfExporter/README.md] `	
 
-1. Make a new folder 
+5. Change directories to the OSX installer directory ` cd synthesis/installer/OSX `
```
Comment (matthew17754) [inline installer/OSX/README.md:21]: Add [] to this path for consistency

**Pull request merged**